src/harness/harness.ts

```diff
@@ -1028,7 +1037,16 @@ module Harness {
 
                         case 'newline':
                         case 'newlines':
-                            newLine = setting.value;
+                            if (setting.value.toLowerCase() === 'crlf') {
+                                options.newLine = ts.NewLineKind.CarriageReturnLineFeed;
+                            } else if (setting.value.toLowerCase() === 'lf') {
+                                options.newLine = ts.NewLineKind.LineFeed;
+                            } else if (setting.value === '\\n') {

```

Comment:
I don't even know why the heck we ever introduced that - I think @danquirk might've added it after we futzed with baselining issues, but I think it would be safe to assume we can use the LF flag here.